appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java

```diff
@@ -194,6 +196,11 @@ private void callDelegateWithOneError(
     byte[] result = null;
     final Double timeoutInSeconds = 10.0;
 
+    Logger logger = Logger.getLogger(VmApiProxyDelegate.class.getName());
+    Level level = logger.getLevel();
+    if (!logger.isLoggable(Level.FINE)) {
+      logger.setLevel(Level.SEVERE);
```

Comment:
The logic here is saying that if the log level has not been set to FINE/FINER/FINEST, then raise it to SEVERE, so that WARNINGs are not logged.  This hides the stacks that are generated by the tests, unless somebody sets one of the debug levels indicating they want to see the detail.